### PR TITLE
React Photo Gallery Component

### PR DESCRIPTION
**Pull request by ericbuth**
@simpixelated

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,7 +1,20 @@
 import React from 'react';
 import { render } from 'react-dom';
+import _ from 'lodash';
+import PhotoGallery from 'photo-gallery';
+
+let photos = _.map (_.range (20), function (num) {
```
Comment (simpixelated) [inline src/index.js:6]: What's the reasoning behind using `let` vs. `const` here?

File: src/index.js [hunk 2]
```diff
@@ -1,7 +1,20 @@
...
+  return {
+    path: 'http://loremflickr.com/640/480?random=' + num
```
Comment (simpixelated) [inline src/index.js:8]: Could you also use a template string here?

File: src/photo-gallery.js [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+import styles from './index.css';
+
+class PhotoGallery extends React.Component {
+
+  constructor (props) {
+    super(props);
+
+    this.state = {
+      photos: this.props.photos
+    };
```
Comment (simpixelated) [inline src/photo-gallery.js:15]: What's the benefit of using `state` here?

**Review by simpixelated — COMMENTED** [review]
Sorry, forgot to submit this yesterday!

File: src/photo-gallery.js (hunk 3 above)
Comment (ericbuth) [reply src/photo-gallery.js:15]: The plan was to use `state` to manage the collection of photos and then `setState()` to update the collection based on deletions and re-orders.

File: src/index.js (hunk 1 above)
Comment (ericbuth) [reply src/index.js:6]: This should be a `const`. I think I did some re-factoring and missed this one.

File: src/index.js (hunk 2 above)
Comment (ericbuth) [reply src/index.js:8]: That would be a slick way of doing it. I was just using the example code that was provided.

**Pull request merged**